internal/characters/charlotte/burst.go

```diff
@@ -0,0 +1,107 @@
+package charlotte
+
+import (
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attacks"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/geometry"
+	"github.com/genshinsim/gcsim/pkg/core/player"
+)
+
+var (
+	burstFrames     []int
+	burstTickFrames = []int{95, 119, 143, 166, 179, 203, 226, 249}
+)
+
+const (
+	burstStart        = 53
+	burstAttackRadius = 6
+	burstHealRadius   = 7
+	burstOffsetX      = 0
+	burstOffsetY      = 4.5
+	burstConsumeDelay = 7
+	burstCD           = 1200
+	burstInitialAbil  = "Still Photo: Comprehensive Confirmation"
+	burstDotAbil      = "Still Photo: Kamera"
+	healInitialMsg    = "Still Photo: Comprehensive Confirmation"
+	healDotMsg        = "Still Photo: Kamera"
+)
+
+func init() {
+	burstFrames = frames.InitAbilSlice(70) // Q -> Walk, Q -> N1
```

Comment:
Q -> N1 should probably be 68 which means a change in the frames arg to the 69f from skill cancel and writing out 70f for the walk cancel (first cancel is 3458**1**-34513 instead of 3458**3**-34513, correct the other trials as well)